src/subclass/widget.rs

```diff
@@ -350,23 +350,25 @@ impl<T: WidgetImpl + ObjectImpl> WidgetImplExt for T {
         unsafe {
             let data = self.get_type_data();
             let parent_class = data.as_ref().get_parent_class() as *mut gtk_sys::GtkWidgetClass;
-            let f = (*parent_class)
-                .button_press_event
-                .expect("No parent class impl for \"button_press_event\"");
-            let ev_glib = glib::translate::mut_override(event.to_glib_none().0);
-            Inhibit(from_glib(f(widget.to_glib_none().0, ev_glib)))
+            if let Some(f) = (*parent_class).button_press_event {
```

Comment:
Can this part be macro?
`call_parent!(button_press_event, f(widget.to_glib_none().0, ev_glib))`

Reply:
No because it looks different depending on what the parent class expects by default.

Reply:
Ok.